### DISCUSSION TITLE
Add placeholder for tagging sound

### DISCUSSION THREAD

**Discussion by NoltoxGit** [Zrips/CMI#6857] [Feature Requests]
Hello !

Since the addition of the /cmi options, I thought that there would be the addition of a placeholder for the tagsound, but apparently it is still not there.

So I suggest to add a placeholder for that!
Example :
`%cmi_user_toggle_tagsound%`
`%cmi_user_togglename_tagsound%`

Ty,
~ @NoltoxGit ~

**Comment by mrfloris** [comment]
To be more complete with the new /cmi options, i completely agree, it would help with visualizing the global options - this one is missing in my opinion.

**Comment by NoltoxGit** [comment]
> Hello !
> 
> Since the addition of the /cmi options, I thought that there would be the addition of a placeholder for the tagsound, but apparently it is still not there.
> 
> So I suggest to add a placeholder for that! Example : `%cmi_user_toggle_tagsound%` `%cmi_user_togglename_tagsound%`
> 
> Ty, ~ @NoltoxGit ~

Helloww,

I'm still looking forward to the addition of this placeholder...

**Reply by NoltoxGit** [reply]
@Zrips 👋

**Comment by mrfloris** [comment]
<img width="1035" alt="Screen-Shot 2022-09-25 at 18 56 03" src="https://user-images.githubusercontent.com/28841349/192155578-fcfacb67-579f-4060-a22d-f198175ea102.png">

It looks like it's been added.
